xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -1612,7 +1625,7 @@ In application code, this field is most commonly used to safely represent a <xre
       <Docs>
         <param name="item">The string to locate in the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />.</param>
         <summary>Returns the zero-based index of the first occurrence of an item in the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />.</summary>
-        <returns>the zero-based index of the first occurrence of <paramref name="item" /> within the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />, if found; otherwise, –1.</returns>
+        <returns>the zero-based index of the first occurrence of <paramref name="item" /> within the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />, if found; otherwise, ?1.</returns>
```

Comment:
?1 -> -1

Reply:
Thanks. I made sure to also fix casing from your suggestion.